tests/unit/amo/components/TestSearch.js

```diff
@@ -73,28 +77,39 @@ describe('<Search />', () => {
     expect(sort.prop('pathname')).toEqual(props.pathname);
   });
 
-  it('does not render a SearchSort when there are no filters', () => {
-    const root = render({ hasSearchParams: false, results: [] });
+  it('does not render a SearchSort when there are no results', () => {
+    const root = render({ count: 0 });
```

Comment:
Sounds good, search is important and it'd be great to get it using real dispatches 👍